packages/datagateway-common/src/table/columnFilters/dateColumnFilter.component.tsx

```diff
@@ -64,17 +69,20 @@ export function updateFilter({
 export const CustomClearButton = (
```

Comment:
Do we need a custom clear button? It seems like the existing one works fine (aka, delete the slot for actionBar, and the slot prop being `["clear"]` still renders a clear button)

Reply:
This button was there from the beginning, so I just left it in. I'll remove it and see if it brakes anything.